modules/standard/IO.chpl

```diff
@@ -4398,6 +4401,19 @@ proc channel.close() {
   if e then this._ch_ioerror(e, "in channel.close");
 }
 
+// TODO -- we should probably have separate c_ptr ddata and ref versions
```

Comment:
Yes, but it would be unrelated to this patch (which is about format() ) and the method in question isn't really user-facing.